packages/fast-tooling/README.md

```diff
@@ -0,0 +1,44 @@
+# FAST Tooling
+
+FAST Tooling is a non-javascript library specific set of utilities to assist in creating web UI.
```

Comment:
This phrasing was a bit awkward intentionally, maybe just "library agnostic"? "non-javascript" made me think this *wasn't* javascript